### ISSUE TITLE
Allow to add existing connected Gerrit URL to another project

### ISSUE THREAD

**Issue by thakurveerendras** [communitybridge/easycla#4392]
**Describe the bug**
Allow to add existing connected Gerrit URL to another project

**To Reproduce**
1. Login to the admin portal
2. Search & open project 1
3. Navigate to easycla -Repository page
4. Connect Gerrit URL 
5. Now search & open another project say 2
6. Navigate to easycla -Repository page
7. Connect & add the same Gerrit to Project 2
 

**Expected behavior**
Should not allow the existing connected Gerrit URL to another project

**Video**

> [GerritConnection.webm](https://github.com/user-attachments/assets/d1f05ac6-6d3a-48be-9f3e-04b716f657dc)

**Comment by thakurveerendras** [comment]
Related PCC ticket: https://github.com/linuxfoundation/lfx-pcc/issues/2461

**Comment by mlehotskylf** [comment]
I believe this is still reproducible. We should be aware if this since we are working on Gerrit support in PCC now.

**Comment by mlehotskylf** [comment]
One of these calls should fail: 

```
POST https://pcc-bff.dev.platform.linuxfoundation.org/development/api/cla-services/cla-group/01af041c-fa69-4052-a23c-fb8c1d3bef24/project/a09P000000DsCE5IAN/gerrit
GET https://pcc-bff.dev.platform.linuxfoundation.org/development/api/cla-services/cla-groups/01af041c-fa69-4052-a23c-fb8c1d3bef24/project/a09P000000DsCE5IAN/gerrit
```

**Comment by lukaszgryglicki** [comment]
According to today's zoom call - this is not a bug but an expected feature?
Pls confirm @mlehotskylf @thakurveerendras ?

**Comment by lukaszgryglicki** [comment]
Ping?

**Comment by lukaszgryglicki** [comment]
Any updates? Everything is correct IMHO and this ticket should be closed.

**Comment by mlehotskylf** [comment]
I agree, according to Eric this is not a bug, I believe we can close it. Further, we may be removing Gerrit onboarding from PCC all together.
@thakurveerendras please let us know if you have any concerns.

**Comment by thakurveerendras** [comment]
Thanks, Closing this ticket as invalid
